packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.7.0+1
+
+* Fix issue: [camera] [crash] Methods marked with @UiThread must be executed on the main thread #72340
```

Comment:
nit: It would be nice if we can update this to form a correct sentence. Maybe something like this:
```suggestion
* Ensure communication from JAVA to Dart is done on the main UI thread.
```

Reply:
Done as you requested